flocker/control/_protocol.py

```diff
@@ -240,6 +243,19 @@ class ClusterStatusCommand(Command):
     response = []
 
 
+class SetNodeEraCommand(Command):
+    """
+    Set the era for a node.
```

Comment:
As in, "Tell the control service that this is the era for a node" or as in, "Tell a node what it's era is"? Please update the docs to clarify.